bot/common/bot/bot.py

```diff
@@ -1,4 +1,9 @@
 import discord
 
-intents = discord.Intents.all()
-bot = discord.Bot(intents=intents)
+
+def get_bot() -> discord.Bot:
```

Comment:
slight convenience refac to allow re-use of bot creation logic outside of main()